includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -610,7 +612,8 @@ static function ( $domain ) {
 		}
 
 		// Optimized Checkout feature flag + setting + whether we are on any of the pages that should not show OC.
-		$should_show_optimized_checkout                 = $this->oc_enabled && $this->is_valid_optimized_checkout_page();
+		// Keep OCS gated to cases where Stripe is the first available gateway.
+		$should_show_optimized_checkout                 = $this->should_use_optimized_checkout_payment_method_layout();
```

PHPMD (2.15.0)

[error] The variable $should_show_optimized_checkout is not named in camelCase. (undefined)

(CamelCaseVariableName)